internal/tweets/explore.go

```diff
@@ -0,0 +1,79 @@
+package tweets
+
+import (
+	"net/http"
+	"strconv"
+
+	"Twitter_like_application/internal/services"
+)
+
+const (
+	badPage    = "invalid page parameter"
+	defPage    = "1"
+	defPerPage = "1"
```

Comment:
probably should be 10 or something greater than one

Reply:
also it would make more sense to use int type for it and not parse the string for the defaults